moveit_msgs/CollisionObject[] objects
